alabaster/__init__.py

```diff
@@ -16,9 +16,8 @@ def update_context(app, pagename, templatename, context, doctree):
 
 
 def setup(app):
-    # add_html_theme is new in Sphinx 1.6+
-    if hasattr(app, "add_html_theme"):
-        theme_path = os.path.abspath(os.path.dirname(__file__))
-        app.add_html_theme("alabaster", theme_path)
+    app.require_sphinx("1.6")
```

Comment:
For some reason this results in a test failure when building Sphinx 6.1.3:
`sphinx.errors.VersionRequirementError: The alabaster extension used by this project needs at least Sphinx v1.6; it therefore cannot be built with this version.`

Using the following build script:
https://github.com/archlinux/svntogit-community/blob/d6ca41cc1d2b0e3a8b9be4752e4a6d02c1f76143/trunk/PKGBUILD

Reply:
Please could you open a new issue? I’ll look to resolve this before a new release of Sphinx or Alabaster.

A

Reply:
#201